generators/server/templates/src/main/resources/config/liquibase/changelog/initial_schema.xml.ejs

```diff
@@ -19,10 +19,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <databaseChangeLog
     xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
-    xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd
-                        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
```

Comment:
Please test this with MSSQL, that's why they were added originally.  https://github.com/jhipster/generator-jhipster/commit/a56a603fcea6a66acce1d7315afebbe01ff070cb

Reply:
You're right I missed this, I prefer to remove the commit related to this modification.
I can optimized it by templating this part and add only needed schema but I feel it will make it more complicated to handle.

Not sur it worth it.

Reply:
nope not worth it

Reply:
So no regrets to remove this commit ;)

I'm waiting for travis to be green and I'll remove the WIP tag ;)

Reply:
So no regrets to remove this commit ;)

I'm waiting for travis to be green and I'll remove the WIP tag ;)

Reply:
So no regrets to remove this commit ;)

I'm waiting for travis to be green and I'll remove the WIP tag.

Reply:
So no regrets to remove this commit ;)

I'm waiting for travis to be green and I'll remove the WIP tag.

Reply:
So no regrets to remove this commit ;)

I'm waiting for travis to be green and I'll remove the WIP tag.

Reply:
So no regrets to remove this commit ;)

I'm waiting for travis to be green and I'll remove the WIP tag.

Reply:
So no regrets to remove this commit ;)

I'm waiting for travis to be green and I'll remove the WIP tag.

Reply:
So no regrets to remove this commit ;)

I'm waiting for travis to be green and I'll remove the WIP tag.